wagtail/contrib/redirects/templates/wagtailredirects/import_summary.html

```diff
@@ -0,0 +1,40 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% block titletag %}{% trans "Summary" %}{% endblock %}
+{% block content %}
+    {% trans "Summary" as summary_str %}
+    {% include "wagtailadmin/shared/header.html" with title=summary_str icon="redirect" %}
```

Comment:
Similar to the other headers, having a consistent main header and adding a subtitle may provide a bit more of a unified feel to this workflow.

```
    {% trans "Import redirects" as header_title %}
    {% trans "Summary" as header_subtitle %}
    {% include "wagtailadmin/shared/header.html" with title=header_title subtitle=header_subtitle icon="redirect" %}
```